helpers/manifest-parser.js

```diff
@@ -139,8 +139,7 @@ function parseIcon(raw) {
   let sizes = parseString(raw.sizes);
   if (sizes.value !== undefined) {
     let set = new Set();
-    sizes.value.split(/\s/).forEach(size => set.add(size.toLowerCase()));
-
+    sizes.value.trim().split(/\s+/).forEach(size => set.add(size.toLowerCase()));
```

Comment:
I can't think of any reasonable interpretation of this that wouldn't trim first and [Chrome appears to agree](https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/Source/core/html/HTMLLinkElementSizesAttributeTest.cpp&sq=package:chromium&type=cs&l=35) :)